### ISSUE TITLE
Can't load it

### ISSUE THREAD

**Issue by lalzy** [bogosortist/cl-sdl2-ttf#21]
**platform:** Windows 10 
**architecture:** Intel x86-64
**Implementation:** CCL
**DLL Path:** "C:\quicklisp\SDL2 DLLS\SDL2_ttf.dll" 

The dll path is set in the path enviornment variable, as all other SDL2 libraries can be run(gfx, image, mixer), it's only ttf that doesn't work. 

**The error I get:** 

```
Subprocess (:PROCESS
            #<EXTERNAL-PROCESS (#1=gcc #2=-o
                                #3=C:/Users/sveny/AppData/Local/cache/common-lisp/ccl-1.11-f96-win-x64/c/quicklisp/dists/quicklisp/software/cffi_0.22.1/libffi/libffi-types__grovel-tmp70D3CAC9.obj
                                ...)[NIL] (ERROR : 2) #x2101EF3B1D>)
 with command (#1# #2# #3# "-c" "-m64"
               "-Ic:/quicklisp/dists/quicklisp/software/cffi_0.22.1/"
               "C:/Users/sveny/AppData/Local/cache/common-lisp/ccl-1.11-f96-win-x64/c/quicklisp/dists/quicklisp/software/cffi_0.22.1/libffi/libffi-types__grovel.c")
 exited with error code 2
   [Condition of type CFFI-GROVEL:GROVEL-ERROR]
```

-- edit: --

Since I had the same problem again after switching computers, and had to re-figure out what the issue was.

The issue is either not having GCC++, or having it for the wrong arcitechture (like x86 instead of x86-64).

So for windows, should get MSYS2 and install MinGW64 through that, as you also need to get libffi through it as well.

So process would be:

1: Download and install MSys2
2: Start MSys2.exe (terminal)
3: in the terminal, type: "pacman -S mingw-w64-x86_64-gcc"
4:  in the terminal, type: "pacman -S mingw-w64-x86_64-libffi"
5:  in the terminal, type: "pacman -S pkg-config"

And you should be good to go.

**Comment by dan-passaro** [comment]
I followed your directions to get cl-sdl2-ttf loading on Windows. I also had to add these directories to the PATH environment variable:
- `C:\msys64\mingw64\bin`
- `C:\msys64\usr\bin`

As well as download the SDL2_ttf.dll file from [the official releases](https://github.com/libsdl-org/SDL_ttf/releases). Thanks for writing up these instructions! I would've been lost trying to get this to work in Windows without them.
